snappass/main.py

```diff
@@ -74,6 +74,9 @@ def clean_input():
     if 'password' not in request.form:
         abort(400)
 
+    if not len(request.form['password']) > 0:
```

Comment:
Should the same change be done to the ttl check to keep things consistent? Even though the ttl values are in a dropdown, someone could still try and inject a blank value.